# Determination - Deterministic rendering environment for music and art
# Copyright (C) 2024 Liu Hao <whiteaxe@tuta.io>
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, version 3.

# This patch removes Ardour's bundled audio export presets. Ardour shows a
# warning every time it tries to load its bundled MP3 presets because we don't
# include all the dependencies Ardour needs to export MP3 files in this image.
# By getting rid of Ardour's bundled export presets we can silence these
# warnings without having to include useless MP3 dependencies.

--- a/wscript
+++ b/wscript
@@ -346,7 +346,7 @@ children = [
         'libs/plugins/a-fluidsynth.lv2',
         #'libs/plugins/a-vapor.lv2',
         # arch independent data
-        'share/export',
+        #'share/export',
         'share/media',
         'share/midi_maps',
         'share/mcp',
